packages/openactive-integration-tests/DEVELOPMENT.md

```diff
@@ -76,16 +76,11 @@ function (configuration, orderItemCriteria, featureIsImplemented, logger, state,
 
     return chakram.wait();
   });
-
-  afterAll(async function () {
-    await state.deleteOrder();
```

Comment:
Based on our earlier conversation, I've removed this from the default advice for new tests

Reply:
@lukehesluke think I mentioned before, but I wonder if we should in fact make this configurable? As it might be that systems want to minimise the number of new orders created, especially in Random mode?

Reply:
That makes sense. So I'll leave this for now (as we'd probably want this configuration option to be exercised in the feature-helper rather than in the tests' afterAll blocks?) and then create an issue for setting this up. Sound good?

Reply:
Sounds great!

Reply:
Issue: https://github.com/openactive/openactive-test-suite/issues/142